### DISCUSSION TITLE
use common sense

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#411] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
![image](https://user-images.githubusercontent.com/118750743/213475046-68091cbb-071f-48a2-a738-b3392162b662.png)
bro on sh1mmer yall are leaving reviews for the chromebook recovery utility... why? this is literally illegal, dont leave evidence for them to track you down and get potentially expelled from school!!!!!

**Comment by Alex2Mayhem** [comment]
People are just stupid.
